pkg/arm/v5/resources.go

```diff
@@ -38,7 +38,11 @@ const (
 	lbKubernetesName                              = "kubernetes" // must match KubeCloudSharedConfiguration ClusterName
 	lbKubernetesOutboundFrontendConfigurationName = "outbound"
 	lbKubernetesOutboundRuleName                  = "outbound"
+	lbKubernetes443RuleName                       = "port-443"
```

Comment:
rename to be consistent with lbKubernetesProbeName443

Reply:
Updated.